src/index.tsx

```diff
@@ -1,16 +1,18 @@
-
 import React from "react";
 import ReactDOM from "react-dom/client";
 import "./index.css";
 import App from "./App";
 import reportWebVitals from "./reportWebVitals";
 import TwoFactorAuthReminders from "./2FA/2FA-Remind";
 import LoggedOutPage from "./pages/loggedOutPage";
-
+import Login from "./pages/Login";
+import Footer from "./Footer";
 const root = ReactDOM.createRoot(document.getElementById("root")!);
 root.render(
   <React.StrictMode>
-    <TwoFactorAuthReminders />
+    {/* <TwoFactorAuthReminders /> */}
+    <Login />
+    <Footer />
```

Comment:
Why was `<TwoFactorAuthReminders />` removed? If we really don't need it, then I think the import above for it should be removed.